source-map-support.js

```diff
@@ -430,29 +430,29 @@ function getErrorSource(error) {
 function printErrorAndExit (error) {
   var source = getErrorSource(error);
 
+  // Ensure error is printed synchronously and not truncated
+  if (process.stderr._handle && process.stderr._handle.setBlocking) {
+    process.stderr._handle.setBlocking(true);
+  }
+
   if (source) {
-    fs.writeSync(2, "\n" + source + "\n");
+    console.error();
+    console.error(source);
   }
 
-  fs.writeSync(2, error.stack + "\n");
+  console.error(error.stack);
   process.exit(1);
 }
 
 function shimEmitUncaughtException () {
-  var origEmit = process.emit;
+  process.on('uncaughtException', function(error) {
```

Comment:
Is there a specific reason for changing this? It will change the before behavior of not exposing us in `process.listeners('uncaughtException')`. I'm not saying that one way or the other is better, but we could potentially break something that depended on it not showing up...

Reply:
Taking over `process.emit` seemed like overkill IMO and a non-standard way of doing it. Not directly related to this PR, however.